source/codegen/metadata/nifgen/functions.py

```diff
@@ -2902,6 +2974,71 @@
         ],
         'returns':'ViStatus'
     },
+    'WriteNamedWaveformI16': {
+        'codegen_method': 'public',
```

Comment:
I dont think we need this codegen method tag here. If it not needed, please remove it

Reply:
Removed from all functions which were added in this PR.